docs/configuring-swc.md

```diff
@@ -95,20 +99,25 @@ While bundling, this option will greatly reduce your file size.
 
 ## jsc.target
 
-Starting from `@swc/core` v1.0.27, you can specify the target environment by using the field.
+Starting from `@swc/core` v1.0.27, you can specify the target environment for output code by using the `jsc.target` field.
+
+This will cause only the necessary transforms for the target environment to be applied when compiling the source code.
+
+For example: to disable transforms for es3/es5/es2015, set the target to `es2016` and only the transforms needed to support a es2016 and later runtime will be applied:
```

Comment:
I am unsure if the default behaviour is to include all transforms (`es3`) or no transforms (`esnext`) ?

Reply:
`swc` has a bug of `target` (due to renaming) and `target: `es2015'` will apply transforms of `es2015`. This will be changed in v2, but I think it worths documenting.

Reply:
I can leave the doc for target as-is for now and we can deal with it separately.